src/prop.rs

```diff
@@ -26,8 +26,7 @@ pub fn store_properties<
     modify_json(&file, |previous_data: &mut Option<S>| {
         match previous_data {
             Some(previous_data) => {
-                // HACK: Find how to handle case where simultaneous writing happens. What is the expected result
-                // Overwrite the data for now
+                // Overwrites data
                 *previous_data = metadata.clone();
```

Comment:
Let's renamed `metadata` into `properties` or `prop` here. We need to merge older and newer version.

The format for merge actually can look weird at first. I propose to use a list for each field.
E.g. `{ title: ["cool website", "amazing website"], description: [ "just one" ] }`

Merging should keep only unique values, i.e. field `description: ["just one", "just one"]` is not valid.

In UI, we'll display only 1 element of each list and hide the rest with some warning icon, so user would be able to review conflicting fields.

See https://github.com/ARK-Builders/ark-components-android/issues/32

Reply:
@kirillt So every properties should be a list and we don't accept plain string for example?

Reply:
A merging function has been added and used for properties